Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Card.css';
+import { CardContainer } from './CardContainer';
+
+export const CardContent = () => {
+	return (
+		<CardContainer
+			link="https://reactjs.org/"
```

Comment:
Nice job! You could also create a data object and pass it as props

Reply:
Alright, thanks I will refactor the code.